### PR TITLE
Matrix operation in c++

### PR DESCRIPTION
**Pull request by somya-kapoor**
#450
Matrix operation: Matrix Multiplication | Recursive, Sum of middle row and column in Matrix, print Lower triangular, print Upper triangular and check idempotent matrix.

### CODE REVIEW

**Comment by gupta2140** [comment]
Please mention in the title the operation that you do on the matrix. The current PR title is vague.

File: Matrix_Operations/C++/To_check_idempotent_matrix.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
```
Comment (jainaman224) [inline Matrix_Operations/C++/To_check_idempotent_matrix.cpp]: Fix file name.

File: Matrix_Operations/C++/Idempotent_Matrix.cpp [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#include <bits/stdc++.h> 
+using namespace std; 
+ 
+const int MAX = 100; 
+ 
+void multiply(int A[][MAX], int res[][MAX],int row,int col) ;
+// Function to check idempotent 
+int checkIdempotent(int A[][MAX],int row,int col) 
+{    
+	int res[MAX][MAX];
```
Comment (jainaman224) [inline Matrix_Operations/C++/Idempotent_Matrix.cpp]: Use 4 space indentation.

File: Matrix_Operations/C++/Idempotent_Matrix.cpp [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  
+// Driver function. 
+int main() 
+{ 
+	int row, col;
+	int A[MAX][MAX];
+	cout << "Enter number of rows and columns of matrix A : ";
+	cin >> row >> col;
+	if (row!=col)
+    {
```
Comment (jainaman224) [inline Matrix_Operations/C++/Idempotent_Matrix.cpp]: indentation fix.

File: Matrix_Operations/C++/Matrix_Sum_of_middle_row_and_column.cpp [hunk 4]
```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
```
Comment (jainaman224) [inline Matrix_Operations/C++/Matrix_Sum_of_middle_row_and_column.cpp]: Use better file name.

**Comment by jainaman224** [comment]
Please fix indentation in all files. Use 4 space indentation instead of a tab.

**Comment by jainaman224** [comment]
Please squash commits.

**Comment by somya-kapoor** [comment]
@jainaman224 Sorry I tried to squash the commit but didn't know what happened.

**Comment by jainaman224** [comment]
@somya-kapoor Have a look at https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git.
If you still face issue. We can discuss on slack.

**Comment by somya-kapoor** [comment]
@jainaman224 I have done the required changes . Please review.

**Pull request merged**